firefly-demo.sh

```diff
@@ -37,6 +37,10 @@ while (! netcat -z 127.0.0.1 "$PORT") ; do sleep 0.1; done
 node_modules/.bin/truffle test test/token/ERC20/ERC20Detailed.test.js &> ../report.txt
 cd ..
 
+#generate coverage data
+cd evm-semantics
+./kevm web3-send "$PORT" 'firefly_getCoverageData' &> ../coverage.json
+
 #close kevm
 cd evm-semantics
```

Comment:
This looks strange. Should there be a `cd ..` after the `./kevm web3-send`?

Reply:
Yes, it should.